master/internal/sproto/jobs.go

```diff
@@ -134,6 +134,11 @@ type UnregisterJob struct {
 // SchedulingState denotes the scheduling state of a job and in order of its progression value.
 type SchedulingState uint8 // CHECK perhaps could be defined in resource manager. cyclic import
 
+// GetNonDaiJobs requests details for Non DAI jobs.
+type GetNonDaiJobs struct {
```

Comment:
Don't think the name of the structure should begin with the verb "Get".  Looking at the description on line 137, this doesn't actually get anything, as it just contains a string. It also has the same name as the function on line 224 in actor_resource_manager.go.  That's a bit confusion.

Reply:
fine to keep it that way for consistency. but usually getter dont even start with `Get` in go so this whole file is weird.